content/terminal/custom-backend/custom-backend.md

```diff
@@ -32,54 +36,145 @@ import TutorialVideo from '@site/src/components/General/TutorialVideo.tsx';
 />
 */}
 
-This is a versatile way to connect any data to the OpenBB Terminal. Whether your API is hosted internally or externally, this method provides a standardized structure that OpenBB Terminal Pro widgets can render.
+## Widget Types
```

Comment:
I think maybe we switch this and create your own backend.  Flow makes more sense - how to make it - what you can make.